k8s/deployments.go

```diff
@@ -59,13 +59,15 @@ type deploy struct {
 }
 
 type DeploymentConfig struct {
-	RevisionHistoryLimit int32 `envconfig:"revision_history_limit" default:"5"`
+	FinishTimeout        time.Duration `split_words:"true" default:"30m"`
+	RevisionHistoryLimit int32         `split_words:"true" default:"5"`
+	StartTimeout         time.Duration `split_words:"true" default:"10m"`
 }
 
 var deploymentConfig DeploymentConfig
 
 func init() {
-	err := envconfig.Process("teresadeploy", &deploymentConfig)
+	err := envconfig.Process("teresa_deploy", &deploymentConfig)
```

Comment:
I would changed it right now as it is undocumented, but to be on the safer side we can change the env vars style for `0.3.0` and document the change. Agree?